docs/src/howto-guides/017_apama_software_management_plugin.md

```diff
@@ -1,70 +1,40 @@
 
+> Note: This plugin expects an Apama installation on the device.
 
+## Install Apama artifacts from Cumulocity
 
-## Download and Install Apama Plugin
+Before an Apama project can be installed on the device using the software management feature in Cumulocity, the project files need to be added to the Cumulocity software repository.
 
-> Note: This plugin expects an apama installation on the device which is managed as a System V service. If Apama is not installed or if the System V init script is not installed, follow the Apama installation instructions [here](https://github.com/thin-edge/thin-edge.io_examples/tree/main/StreamingAnalytics#setup-and-configuration) before installing/using the apama plugin.
+There is a naming convention that you need to follow while creating software entries for Apama artifacts in the software repository.
 
-Apama plugin is delivered as a debian package which can be downloaded from the releases section of thin-edge.io repository on [GitHub](https://github.com/thin-edge/thin-edge.io/releases).
+For Apama projects:
 
-Example command for downloading the `0.5.0` version of the plugin for `armhf` architecture (Adjust the version and architecture values as per your requirements):
-
-```shell
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.5.0/tedge_apama_plugin_0.5.0_armhf.deb
-```
-
-Once the debian package is downloaded, install it with the `dpkg` command:
-
-```shell
-sudo dpkg -i tedge_apama_plugin_0.5.0_armhf.deb
-```
-
-Once the plugin is installed on the device, you can install apama projects and mon files using Cumulocity software management feature.
-
-## Install Apama artefacts from Cumulocity
-
-Before an apama project or a mon files can be installed on the device using the software management feature in Cumulocity, these project files or monitor files need to be added to the Cumulocity software repository first.
-
-There's some naming convention that you need to follow while creating software entries for apama artefacts in the software repository.
-
-For apama projects:
-
-1. Name must be suffixed with `::project` as in `my-demo-project::project`
-2. Version must be suffixed with `::apama` as in `1.0::apama` or just `::apama` if you don't need a version number.
-3. The uploaded binary must be a `zip` file that contains the `project` directory. If a directory named `project` is not found at the root level in the zip, it's considered invalid.
+1. The name must be suffixed with `::project` as in `my-demo-project::project`.
+2. The version must be suffixed with `::Apama` as in `1.0::Apama` or just `::Apama` if no  version number is necessary.
+3. The uploaded binary must be a `zip` file that contains the `project` directory. If a directory named `project` is not found by the plugin at the root level in the zip, it is considered invalid.
 
 ![Add new apama project in Software Repository](./images/apama-plugin/apama-project-c8y-software-repository.png)
 
-For apama monitor file:
```

Comment:
Why was this section on injecting `mon` files removed? Doesn't the new version of the plugin support this anymore?

Reply:
Yes, the new version will not support this.